README.md

```diff
@@ -502,6 +502,7 @@ Below are some examples
 * [CoffeeScript](https://www.npmjs.com/package/coffee-script): `--compiler coffee:coffee-script/register`
 * [TypeScript](https://www.npmjs.com/package/ts-node): `--compiler ts:ts-node/register`
 * [Pogo](https://www.npmjs.com/package/pogo): `--compiler pogo:pogo`
+* [LiveScript](http://livescript.net): `--compiler ls:LiveScript`
```

Comment:
I have updated the PR to lowercase (OS X with its case insensitive filesystem didn't catch this issue). Feel free to close this PR if you want to go the route of less examples. I guess CoffeeScript or Babel would be the most commonly used one here.